crates/autopilot/src/domain/settlement/fees.rs

```diff
@@ -0,0 +1,14 @@
+//! There are two types of fees in the system: gas cost fees and protocol fees.
+//!
+//! Gas cost fees are fees that are paid to a network for the onchain
+//! execution of the settlement. These fees are paid in ETH and are calculated
+//! based on the amount of gas used by the transaction. The gas cost fees are
+//! determined by solvers.
+//!
+//! Protocol fees are fees that are paid to the protocol for it's services.
+//! These fees are paid in the native token of the protocol and are determined
+//! by the protocol.
+
+// TODO
+
+pub struct Fees {}
```

Comment:
Can't we reuse the existing domain/auction model here? What will be different about fees in the context of the settlement?

Reply:
There is nothing to reuse atm. Autopilot domain fee contains, let's say, the input for protocol fee calculation. This module is supposed to contain the output - executed values.

Reply:
I see. I think like surplus this should be directly exposed on the domain settlement struct (and the computation should be private to that module).

Reply:
killed the file. moved surplus / fee / score calculation inside Settlement.